clone_kit.gemspec

```diff
@@ -22,16 +22,21 @@ Gem::Specification.new do |spec|
   spec.executables   = spec.files.grep(%r(^exe/)) { |f| File.basename(f) }
   spec.require_paths = ["lib"]
 
-  spec.add_runtime_dependency "redis"
   spec.add_runtime_dependency "activesupport", "> 3.0.0" # For core ext Array#wrap and Object#blank?
```

Comment:
If we use `arel_attributes_with_values_for_create` then this requirement should be changed to `> 4.0.2`, '< 5.0'